opm/parser/eclipse/EclipseState/Tables/SimpleTable.cpp

```diff
@@ -153,6 +153,16 @@ void SimpleTable::checkMonotonic(const std::string& columnName,
     }
 }
 
+void SimpleTable::checkRange(const std::string& columnName)
```

Comment:
The range `[0,1]` is not *that* canonical; rename the method `assertUnitRange()` - or alternatively take limits as input arguments to the method.